lib/preview/createRoutes.js

```diff
@@ -0,0 +1,36 @@
+const { Router } = require('express');
+const cors = require('cors');
+const { buildIndexRouteHandler } = require('./routes/index.js');
+const { buildModuleRouteHandler } = require('./routes/module.js');
+const { buildTemplateRouteHandler } = require('./routes/template.js');
+const { buildMetaRouteHandler } = require('./routes/meta.js');
+
+const { buildProxyRouteHandler } = require('./routes/proxyPathPageRouteHandler.js');
+const { buildProxyPageRouteHandler } = require('./routes/proxyPageRouteHandler.js');
+const { proxyPathPageResourceRedirect } = require('./routes/proxyPathPageResourceRedirect.js');
+const { proxyPageResourceRedirect } = require('./routes/proxyPageResourceRedirect.js');
+
+const createPreviewServerRoutes = async (sessionInfo) => {
```

Comment:
Might be nice to add some comments in here to explain what each of these handlers is doing. It's a little tough to read through this and follow what's happening.